changelog/fix_handle_ignored_columns_from_mixins.md

```diff
@@ -0,0 +1 @@
+* [#601](https://github.com/rubocop/rubocop-rails/pull/601): Handle ignored_columns from mixins. ([@tachyons][])
```

Comment:
Can you adjust to mention `Rails/UnusedIgnoredColumns`?

Reply:
Done